CHANGELOG.md

```diff
@@ -1,14 +1,23 @@
+# 3.0.0
+
+### Major
+
+* Switched to Wren group and artifact ids
+
+
 # 2.0.0
 
 ## Changes
```

Comment:
Should be used either nowhere or everywhere.

Reply:
I would rather have Java 17 support as a separate PR as this one is big as it is already.

Reply:
We need to decide what to do with this file after merging the project in wrensec-commons.